cpp/include/utilities/cython.hpp

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <graph.hpp>
+// #include <experimental/graph_view.hpp>
+#include <raft/handle.hpp>
+
+namespace cugraph {
+namespace cython {
+
+// FIXME: use std::variant instead of a union if possible
+// FIXME: add both CSRView and graph_type_t objects for easier testing during
+//        the transition
+union graphUnion {
+  graphUnion() {}
+  GraphCSRView<int, int, float> GraphCSRViewFloat;
+  GraphCSRView<int, int, double> GraphCSRViewDouble;
+};
+
+enum weightTypeEnum { floatType = 0, doubleType = 1 };
```

Comment:
Please consider using scoped enumeration 
```
enum class weightTypeEnum : int { floatType = 0, doubleType = 1};
```
More readable, provides better name hiding (scoping), and stronger typing.

Reply:
done in latest commit